src/network/network_admin.cpp

```diff
@@ -333,11 +332,27 @@ NetworkRecvStatus ServerNetworkAdminSocketHandler::SendCompanyInfo(const Company
 	p->Send_uint8 (CeilDiv(c->months_of_bankruptcy, 3)); // send as quarters_of_bankruptcy
 
 	for (auto owner : c->share_owners) {
-		p->Send_uint8(owner);
+	  p->Send_uint8(owner);
```

Comment:
OpenTTD uses tabs to to align statement. I think you replace them with spaces here.